docs/theme/cta.html

```diff
@@ -1,47 +1,46 @@
+        with premium support and additional, enterprise-grade features. It is for organizations
+        that need to secure critical production infrastructure and meet
+        compliance and audit requirements.
+      </p>
 
-    <!-- TELEPORT ENTERPRISE -->
-    <div class="col-6_md-12">
-      <div class="paper">
-        <h3 class="text-title is-uppercase is-bold is-plum">Teleport Enterprise</h3>
-        <p class="text-body">
-          Teleport Enterprise is built around the open-source core,
-          with premium support and additional, enterprise-grade features. It is for organizations
-          that need to secure critical production infrastructure and meet
-          compliance and audit requirements.
-        </p>
-
-        <a class="is-block button button-primary is-large l-space-top-4" href="/teleport/demo/">Demo Teleport Enterprise</a>
-      </div>
+      <a class="is-block button button-primary is-large l-space-top-4" href="/teleport/demo/">Demo Teleport Enterprise</a>
     </div>
+  </div>
 
-    <!-- TELEPORT COMMUNITY -->
-    <div class="col-6_md-12">
-      <div class="paper">
-        <h3 class="text-title is-plum">
-          Teleport Community
-        </h3>
+  <!-- TELEPORT COMMUNITY -->
+  <div class="col-6_md-12">
+    <div class="paper">
+      <h3 class="text-title is-plum">
+        Teleport Community
+      </h3>
 
-        <p class="text-body">
-          Teleport Community provides modern SSH best practices out
-          of the box for managing elastic infrastructure. Teleport Community
-          is open-source software that anyone can download and install for free.
-        </p>
+      <p class="text-body">
+        Teleport Community provides modern SSH best practices out
+        of the box for managing elastic infrastructure. Teleport Community
+        is open-source software that anyone can download and install for free.
+      </p>
 
-        <p class="text-body is-center l-space-top-2">
-          <iframe class="widget-github"
-            frameborder="0"
-            width="95px"
-            height="20px"
-            scrolling="0"
-            src="https://ghbtns.com/github-btn.html?user=gravitational&amp;repo=teleport&amp;type=star&amp;count=true">
-          </iframe>
-        </p>
+      <p class="text-body is-center l-space-top-2">
+        <iframe class="widget-github"
+          frameborder="0"
+          width="95px"
+          height="20px"
+          scrolling="0"
+          src="https://ghbtns.com/github-btn.html?user=gravitational&amp;repo=teleport&amp;type=star&amp;count=true">
+        </iframe>
+      </p>
 
-        <a class="is-block button button-secondary button-border is-large" href="/teleport/download/">Download Teleport Community</a>
+      <a class="is-block button button-secondary button-border is-large" href="/teleport/download/">Download Teleport Community</a>
 
-      </div>
     </div>
   </div>
+  </div>
```

Comment:
The indenting is a bit off :neckbeard: 
https://www.freeformatter.com/html-formatter.html